web/packages/build/webpack/webpack.base.js

```diff
@@ -78,7 +78,7 @@ const configFactory = {
       return {
         test: /\.svg$/,
         type: 'asset/inline',
-        exclude: /node_modules/,
+        exclude: /[\\/]node_modules[\\/]/,
```

Comment:
Is this change necessary? As this will technically introduce a regression from the previous...

| old | new | path | 
|---|---|---|
| ✔️  | ✖️   | node_modules/foo/bar |
| ✔️  | ✔️  | /node_modules/foo/bar |
| ✔️  | ✔️  | foo/bar/node_modules/foo/bar |
| ✔️  | ✔️   | /foo/bar/node_modules/foo/bar |

Reply:
Webpack seems to match on absolute paths, can be verified via e.g.:

```diff
diff --git a/e b/e
index 04605151e7..db056f5f5f 160000
--- a/e
+++ b/e
@@ -1 +1 @@
-Subproject commit 04605151e7be43fb6831764153cf485e3a6a493f
+Subproject commit db056f5f5fbf96d1fd99c947d8f5be9070715363
diff --git a/web/packages/build/webpack/webpack.base.js b/web/packages/build/webpack/webpack.base.js
index 2806ab488d..ff7b79603d 100644
--- a/web/packages/build/webpack/webpack.base.js
+++ b/web/packages/build/webpack/webpack.base.js
@@ -104,7 +104,10 @@ const configFactory = {
     jsx() {
       return {
         test: /\.(ts|tsx|js|jsx)$/,
-        exclude: /[\\/]node_modules[\\/]/,
+        exclude: (f) => {
+          console.log(f)
+          return f.match(/[\\/]node_modules[\\/]/)
+        },
         use: [
           {
             loader: 'babel-loader',
```

which outputs:

```
/home/justinas/gravitational/teleport/web/packages/teleport/src/boot.tsx
```

This is how #21662 happened in the first place (the dir containing the `assets` substring was above the repo root).

So there can never be a bare `node_modules` at the start (will always be at least `C:\node_modules` or `/node_modules`).